### PR TITLE
Camera communication

### PR DESCRIPTION
**Pull request by bvanwijk**
Created the IP camera communication module. You can now send commands to the camera's to control: zoom, movement, focus, iris and to get the stream link.
The commands are tested with a mocked server to verify the righ requests are made.

### CODE REVIEW

**Comment by Kingdorian** [comment]
The camera interface looks really nice, the Ipcamera class is a bit to large I think. I think this can be merged but we should adress the size of the Ipcamera class in the future.

File: cameramodule/src/main/java/com/benine/ipcameracontrol/Ipcamera.java [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+
+
+/**
+ * Class to communicate with an IP Camera.
+ * @author Bryan
+ */
+
+public class Ipcamera implements Camera {
+
+  String ipadres;
```
Comment (jochem725) [inline cameramodule/src/main/java/com/benine/ipcameracontrol/Ipcamera.java]: Shouldn't this be ipaddress?

**Comment by jochem725** [comment]
Looks nice, I've placed a small comment regarding a variable name. After this is fixed, I'll agree with merging.

**Comment by NaomidR** [comment]
The IPcamera class should indeed become shorter in the next sprint. But for now I agree with merging.

**Comment by egedebruin** [comment]
Looks good, is it correct that the .checkstyle files belong to the backend and camera module?

**Pull request merged**